lib/track.js

```diff
@@ -217,7 +217,7 @@ Track.prototype.paste = function(e) {
   if (t) {
     // Customisable logic for identifying pan and sensitive fields
     var custom = this.classifyPaste(e);
-    panField = t.hasAttribute('data-rvn-pan') || (custom && custom.pan);
+    panField = t.hasAttribute('data-rvn-pan') || !!(custom && custom.pan);
```

Comment:
@icio What do you think of this? I know the `!!` can be a polarising one

Reply:
I'm not fussed either way.

Curiously, terser will convert this into `!(!F||!F.pan)` during minifcation. If you went with just `(custom && custom.pan)` terser will drop the brackets so we save 4 characters... but that's considerably more pedantic about filesize than I have ever been here.

We're only passing it to an `if` statement, where truthyness is sufficient, rather than exactly `true`, so I don't find this adds much. `!!` is my favourite way to do it when I really want a boolean though.

Reply:
Ok, I'll remove it and save us the 4 unnecessary characters 😅